### ISSUE TITLE
Create a Release PR workflow

### ISSUE THREAD

**Issue by lordrip** [KaotoIO/kaoto-next#669]
### Context
To improve the release process, we should create a new workflow (or change the existing one):

1. [ ] Create a workflow that asks for the version that we want to release.

2. [ ] That workflow creates a new PR changing all `package.json` versions, maybe the `pom.xml` files as well, and updating the changelog.md divided by features, bug fixes, and chores PRs.

3. [ ] Once that PR is merged, we create a draft release by hand, or if can be automated, do so

4. [X] Once the tag is created, a workflow is automatically dispatched to make the release using the package.json version https://github.com/KaotoIO/kaoto-next/pull/668

**Comment by lhein** [comment]
isn't that last point already tackled by the work @oscerd has done?

**Comment by oscerd** [comment]
Everything is done. The only missing piece is bridging between kaoto-next and kaoto-operator jobs.
